Dockerfile

```diff
@@ -1,10 +1,24 @@
 ARG APP_INSIGHTS_AGENT_VERSION=2.5.1
-FROM hmctspublic.azurecr.io/base/java:openjdk-11-distroless-1.2
+FROM hmctspublic.azurecr.io/base/java:openjdk-11-distroless-1.4 as base
 LABEL maintainer="https://github.com/hmcts/ethos-repl-docmosis-service"
 
 COPY lib/AI-Agent.xml /opt/app/
 COPY build/libs/ethos-repl-docmosis-service.jar /opt/app/
 
+FROM debian:10 AS builder
+
+USER root
+RUN apt update
+RUN apt install --yes libharfbuzz-dev
+USER hmcts
+
+FROM base
+
+COPY --from=builder /usr/lib/x86_64-linux-gnu/libharfbuzz.so.0 /usr/lib/x86_64-linux-gnu/libharfbuzz.so.0
```

Comment:
Ideally add a comment linking to upstream issue so that this can be removed when fixed

Reply:
We'll have a Tech debt ticket for that, @timja, linking it to this PR.